lib/posix-environ/environ.c

```diff
@@ -183,3 +186,44 @@ char **__environ = __init_env;
 __weak_alias(__environ, ___environ);
 __weak_alias(__environ, _environ);
 __weak_alias(__environ, environ);
+
+/* Find the first occurrence of a variable within __init_env */
+static char **find_vnpos(const char *varname, size_t varname_len, char **end)
```

Comment:
Ditto `__sz`

Reply:
Because the other functions (`setenv`, `getenv`, ...) in this lib (that were ported from musl) use libc types, I'd like to keep it here as well, so that we are consistent within this library.